app/views/layouts/application.html.erb

```diff
@@ -3,12 +3,50 @@
   <head>
     <title>Attendion</title>
     <%= csrf_meta_tags %>
-
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.css">
```

Comment:
bootstrap wird anders in Rails eigebunden. Schauen Sie dazu die Beispielanwendung an.